tests/www/views/test_views_tasks.py

```diff
@@ -993,3 +993,220 @@ def test_graph_view_doesnt_fail_on_recursion_error(app, dag_maker, admin_client)
         url = f'/dags/{dag.dag_id}/graph'
         resp = admin_client.get(url, follow_redirects=True)
         assert resp.status_code == 200
+
+
+def test_task_instances(app, dag_maker, admin_client):
+    """Test task_instances view."""
+    resp = admin_client.get(
+        f'/object/task_instances?dag_id=example_bash_operator&execution_date={DEFAULT_DATE}',
+        follow_redirects=True,
+    )
+    assert resp.status_code == 200
+    payload = resp.json
+    for v in payload.values():
+        assert v.pop('updated_at')
```

Comment:
Is there a way to mock the server_default, for `updated_at` so I can assert the entire response ?

Reply:
I pushed a commit to do this.

Reply:
thanks @uranusjr, much cleaner this way. I naively froze time on the test, which had no effect since fixture init happens before 🤦‍♂️